src/libraries/sendtohelixhelp.proj

```diff
@@ -98,22 +98,20 @@
     <HelixPreCommand Condition="'$(Scenario)' == 'WasmTestOnBrowser'" Include="export XHARNESS_COMMAND=test-browser" />
     <HelixPreCommand Include="export XHARNESS_DISABLE_COLORED_OUTPUT=true" />
     <HelixPreCommand Include="export XHARNESS_LOG_WITH_TIMESTAMPS=true" />
+    <HelixPreCommand Include="export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1" />
```

Comment:
Just curious, why is this needed here?

Reply:
First time experience does the cert installation too, plus some text dump. So it's not necessary.

Reply:
It might be better to add it here: https://github.com/dotnet/arcade/blob/c336c9622eafce2b9bf3c4c164a4c0b57703aa5e/src/Microsoft.DotNet.Helix/Sdk/tools/dotnet-cli/DotNetCli.targets#L18

Reply:
but arcade is not sure we have SDK, so it would have to be conditional

Reply:
runtime doesn't have the cert tool only SDK does. you can't `dotnet dev-certs https` on runtime without failure.
we could remove both DOTNET_SKIP_FIRST_TIME_EXPERIENCE and `dotnet dev-certs https` from here and the cert would be installed on SDK first run. But it would be implicit side effect, rather than explicit action. Is that OK ?

Reply:
$(TestPackages) is also setting DOTNET_SKIP_FIRST_TIME_EXPERIENCE, so we would have to avoid clash.

Reply:
I meant, add the cert stuff in runtime as you plan to.

I was only talking about `DOTNET_SKIP_FIRST_TIME_EXPERIENCE` that it could go into Arcade too